components/SwitchNetwork/SwitchNetwork.tsx

```diff
@@ -0,0 +1,94 @@
+import { ethers, BigNumber } from 'ethers';
```

Comment:
The component is a modal so it should be named `SwitchNetworkModal`

Reply:
Done